spec/ffi/callback_spec.rb

```diff
@@ -866,6 +866,11 @@ module LibTestFiddle
       extern 'void testClosureVrV(void *fp)'
     end
 
+    after :all do
+      GC.start
+      expect(ObjectSpace.each_object(Fiddle::Closure) {}).to eq(0)
```

Comment:
Given these 2 lines are extremely slow, please only do them if `Process.respond_to?(:fork)` (i.e. it shouldn't be done on TruffleRuby, JRuby and Windows).

Reply:
Extremely slow is probably a bit exaggerated. But it should help with the test failures on JRuby I guess, so thx for the hint

Reply:
Well depending on the heap, a full GC can take seconds, and iterating the ObjectSpace even more.
I suppose the heap is relatively small in those tests so it's likely fine enough for testing.
But `ObjectSpace.each_object` shouldn't be used in any production code because it's way too slow.

Reply:
Oh now I realized that you have meant to hide both lines behind the condition and I was focused on `ObjectSpace` given the failures. Let me update the PR ☝️